### PR TITLE
Spring security implemented

### PR DESCRIPTION
**Pull request by AlinaOryshak**

### CODE REVIEW

File: src/main/java/mate/academy/spring/dto/request/CinemaHallRequestDto.java [hunk 1]
```diff
@@ -1,6 +1,9 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.Min;
+
 public class CinemaHallRequestDto {
+    @Min(10)
     private int capacity;
     private String description;
```
Comment (HryhorashPavlo) [inline src/main/java/mate/academy/spring/dto/request/CinemaHallRequestDto.java:8]: i think better to add @NotNull for field

File: src/main/java/mate/academy/spring/dto/request/MovieRequestDto.java [hunk 2]
```diff
@@ -1,6 +1,11 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.Max;
+import javax.validation.constraints.NotNull;
+
 public class MovieRequestDto {
+    @NotNull
+    @Max(50)
     private String title;
     private String description;
```
Comment (HryhorashPavlo) [inline src/main/java/mate/academy/spring/dto/request/MovieRequestDto.java:10]: same here

File: src/main/java/mate/academy/spring/dto/request/MovieSessionRequestDto.java [hunk 3]
```diff
@@ -1,9 +1,15 @@
...
+import javax.validation.constraints.Positive;
 
 public class MovieSessionRequestDto {
+    @NotNull
+    @Positive
     private Long movieId;
+    @NotNull
+    @Positive
     private Long cinemaHallId;
     private LocalDateTime showTime;
```
Comment (HryhorashPavlo) [inline src/main/java/mate/academy/spring/dto/request/MovieSessionRequestDto.java:14]: same

**Review by HryhorashPavlo — APPROVED** [review]

**Review by boroda4436 — CHANGES_REQUESTED** [review]
Please resolve git conflicts

**Pull request closed without merging**